### DISCUSSION TITLE
chess pieces on Github from a program call how program a chess game in python

### DISCUSSION THREAD

**Discussion by cope61** [community/community#124163] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

i'm looking for some chess pieces that surpose  be here on github to help build this program chess game in python where can i find this folder to down load these images to help in building this program?

**Comment by CopperEagle** [comment]
Hey there @cope61 

I found some good ones [here](https://github.com/ImKadaga/chess-pieces/tree/master/chess_kaneo_midnight), where the bishop looks like this: 
![wB](https://github.com/community/community/assets/137458753/8125ad69-419c-4cac-9c34-2108a52ee6b1)

Boards are also in that repository.
